### DISCUSSION TITLE
[Question] IDK How to ListBox like Tkinter

### DISCUSSION THREAD

**Discussion by bbkmg256** [flet-dev/flet#3065] [Q&A] [answered]
### Question

Well, I'm starting to see what's up with Flet and I don't know anything about it, I'm reading the documentation and such, but I'm running into a problem which I don't know if it's that the documentation, the framework or the widgets don't consider it, or there is some way that I'm not seeing that can be achieved to do what I need. The fact is that I want to make a "list box" similar to the one found in Tkinter, but I don't know which widget would be the most correct to implement, or if there is one that is exactly that, does anyone know?

![captura_xd](https://github.com/flet-dev/flet/assets/132410342/37ae2a62-1f69-4e9a-8245-7e8cbb6ee45b)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by abdokhalil07** [comment]
https://flet.dev/docs/controls/textfield#multiline-textfields
![text](https://github.com/flet-dev/flet/assets/94837888/4875f3e6-da87-4967-9569-fd6672d7c2e1)

**Reply by abdokhalil07** [reply]
but itn't work like tkinter or kivy

**Comment by MrDIDJoy** [comment, marked as answer]
You can just try to create a "Large View" and insert a text control or other controls into it.

You can find more here: [https://flet.dev/docs/cookbook/large-lists/#listview]
I hope I helped you

**Reply by bbkmg256** [reply]
Hello, yes, I had already seen that component, but I thought there would be another one that was more similar, even so I think I'm going to opt for that and see if I can implement what I want to do, thanks for the help bro :)
